### ISSUE TITLE
Add translations

### ISSUE THREAD

**Issue by oliverlockwood** [cyclestreets/android#171]
CycleStreets is currently available only in English.  It would be great to support other languages.

The following strings files all need localisation:
-  https://github.com/cyclestreets/android/blob/master/cyclestreets.vNext/src/main/res/values/strings.xml
-  https://github.com/cyclestreets/android/blob/master/libraries/cyclestreets-core/src/main/res/values/strings.xml
-  https://github.com/cyclestreets/android/blob/master/libraries/cyclestreets-core/src/main/res/values/arrays.xml
-  https://github.com/cyclestreets/android/blob/master/libraries/cyclestreets-fragments/src/main/res/values/strings.xml
-  https://github.com/cyclestreets/android/blob/master/libraries/cyclestreets-track/src/main/res/values/strings.xml
-  https://github.com/cyclestreets/android/blob/master/libraries/cyclestreets-view/src/main/res/values/strings.xml

For each of the files described in the header comment of this PR, you will need to put a file named the same but placed in a corresponding `values-XX` directory that you create.  `XX` corresponds to the country code you're translating into.

A couple more details:
-  Only the _values_ of XML objects should be translated.  The `name` attributes should _not_ be translated as these are what the app uses to identify the text entry.
-  Any entry marked with the XML attribute `translatable=false` should, as the attribute suggests, not be translated - and in fact must not be included in the localised files at all, otherwise the builds will fail.

**Please check for errors locally, by running `./gradlew lint` or similar, to ensure that there are no errors before you submit your PR, as this will save everyone from wasted time.  If for whatever reason you can't do this, please ensure you check the status of automated build that runs when your PR is raised, and fix any errors promptly.**

**Comment by auyer** [comment]
I can translate do Pt-BR

**Comment by auyer** [comment]
I can help with Pt-BR

**Comment by GiORg1** [comment]
Hello

I can translate them in Russian. Can you give me exact instructions how translation must be added?

**Comment by LauraLaureus** [comment]
I can translate into Spanish, and also need accurate instructions. Only the strings between the tags or the name attribute as well?

**Comment by TroyCode** [comment]
I can do this in zh-TW, and also need instruction

**Comment by mvl22** [comment]
Thanks everyone for these offers of help!

In the first instance, please could you each create an account on Transifex, and then sign up at:
https://www.transifex.com/cyclestreets/cyclestreets-android/

and we will get the translations going there shortly.

**Comment by LauraLaureus** [comment]
Done, however I would like to know in which way will we get the pull request needed for the HacktoberFest

**Comment by oliverlockwood** [comment]
Hi @LauraLaureus @TroyCode @GiORg1 - thank you for your offers to submit translations.

Since we haven't yet fully set up our Transifex integration, we're happy for you to directly submit  translations PRs for now.  The instructions in the "How to Create Alternative Resources" section of https://developer.android.com/guide/topics/resources/localization.html document the Android standards describing what we need.

In summary, for each of the files described in the header comment of this PR, you will need to put a file named the same but placed in a corresponding `values-XX` directory that you create.  `XX` corresponds to the country code you're translating into.

A couple more details:
-  Only the *values* of XML objects should be translated.  The `name` attributes should *not* be translated as these are what the app uses to identify the text entry.
-  Any entry marked with the XML attribute `translatable=false` should, as the attribute suggests, not be translated.

**Comment by AlexandreInsua** [comment]
I can help with spanish (Spain).

**Comment by LauraLaureus** [comment]
@AlexandreInsua I'm already on it

**Comment by GiORg1** [comment]
I've added Russian translation with PR #174. Please review and accept it

**Comment by oliverlockwood** [comment]
@AlexandreInsua when @LauraLaureus has raised her PR, if you're able to review her submission from a language perspective that would be great, as I'm not a Spanish speaker!  Second opinions always help 😃

**Comment by vineyugave** [comment]
PR for hi-IN created : https://github.com/cyclestreets/android/pull/176

**Comment by aybuke** [comment]
I can help with Turkish

**Comment by vineyugave** [comment]
PR for mr-IN created : #180

**Comment by florian-f** [comment]
https://github.com/cyclestreets/android/pull/182

**Comment by oliverlockwood** [comment]
We now have a bunch of translations, and can iterate from here.  Closing this issue for now.

**Comment by mvl22** [comment]
Thanks everyone for these translations, and to Oliver for bringing this all together.

**Comment by mvl22** [comment]
Oliver, can you remind me how this is enabled in the app - does it just pick up the language from the phone's OS automatically? Is there any ability to change it within our own settings?

**Comment by oliverlockwood** [comment]
@mvl22 I believe it picks up the language from the phone's OS.  I just changed my language to Spanish and proved that, see the following screenshots.

Admittedly it looks like "Tap map to set Start" has not been localised (I've raised #211 to cover this), but the rest looks plausible.

![screenshot_20180604-083642](https://user-images.githubusercontent.com/6017680/40904612-327e1f92-67d3-11e8-86cd-4befb860ab38.png)
![screenshot_20180604-083635](https://user-images.githubusercontent.com/6017680/40904613-3294fc44-67d3-11e8-8ef8-e80164ff8237.png)
![screenshot_20180604-083630](https://user-images.githubusercontent.com/6017680/40904614-32ac114a-67d3-11e8-8dfe-61dee6ff0a0f.png)
